### ISSUE TITLE
<InfoIcon> should be focusable

### ISSUE THREAD

**Issue by sidoruk-sv** [wix/wix-style-react#6314]
# ✨ Feature Request

### 🔦 Scope

For now when you try to focus the `<InfoIcon/>` with `Tab` key you will miss it and you can not select it.

For a11y purposes: if we are talking about screen readers it should open up the tooltip on focus and read out loud the text inside the tooltip.

But when we have non-selectable `<InfoIcon/>` there is no way to make this happened.

### 💁 Explanation

When user tap `Tab` key he will see focus on the `<InfoIcon />`, when he will tap `Return` key, after that, tooltip content should be shown.

### 🐾 Possible solution 

...

### 👀 Severity

- Low

**Comment by EgoziE** [comment]
Moved to Jira - closing

**Comment by sidoruk-sv** [comment]
Just for cross-linking:
https://jira.wixpress.com/browse/DSM-1472
